src/main/java/core/org/akaza/openclinica/bean/extract/odm/ClinicalDataReportBean.java

```diff
@@ -261,7 +261,8 @@ public void addNodeClinicalData(boolean header, boolean footer) {
                                     && se.getExportFormData().get(0).getEventDefinitionCrf().getStatusId() != Status.DELETED.getCode()) {
 
                                 // ***************** OpenClinica:Link REMOVE EVENT **************
-                                if (studyEvent.getRemoved()==null || (studyEvent.getRemoved()!=null && !studyEvent.getRemoved())) {
+                                if ((studyEvent.getRemoved()==null || (studyEvent.getRemoved()!=null && !studyEvent.getRemoved()))
```

Comment:
Since we are doing a lot of these checks now, I just want to point out that this logic can be simplified to:
BooleanUtils.isFalse(studyEvent.getRemoved())

Reply:
done